spec/factories/task.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :task do
+    name { Faker::Lorem.sentence }
+    author { Faker::FunnyName.name }
+    description { Faker::Quote.yoda }
+    edition { create(:edition) }
```

Comment:
Because the edition is optional, we can have tasks that are not used by a specific edition, we may consider moving this to a separate trait, eg. `with_edition`.